data-raw/data.R

```diff
@@ -85,21 +85,106 @@ translations <- dplyr::bind_rows(
   dplyr::tibble(
     id = "ui_adjustLabel",
     english = "Shift label",
-    french = "Ajuster l'étiquette"
+    french = "étiquette de changement"
```

Comment:
```suggestion
    french = "Étiquette de changement"
```

Do we need this capitalized?

Reply:
yes good catch